tests/test_fastapi_apps.py

```diff
@@ -187,16 +186,6 @@ def test_non_existent_thing(client: TestClient, session: Session):
     assert response.status_code == 404
 
 
-# TODO: re-enable this
-@pytest.mark.skipif(os.environ.get("CI") is not None, reason="Only run this test manually, not in CI.")
-def test_get_thing_all_profiles(client: TestClient, session: Session):
-    response = client.head(f"/thing/{TEST_IGSN}")
-    assert response.status_code == 200
-    # The head method is a profiles request, asking the server for all the profiles the thing supports.  The response
-    # contains the profiles in the links section of the response, so assert on that.
-    assert len(response.links) > 0
```

Comment:
This was another unrelated failure after updating dependencies.  It seems like things changed in the new urllib and the head request is broken on their test client.